packages/blade/src/components/Input/SelectInput/SelectInput.tsx

```diff
@@ -107,7 +107,9 @@ const _SelectInput = (
 
   const selectValues = (valuesToSelect: string | string[]): void => {
     if (options.length > 0) {
-      if (typeof valuesToSelect === 'string') {
+      if (isEmpty(valuesToSelect)) {
```

Comment:
A minor helpful comment that we treat empty string `''` as a way to clear input.